code/mob/living/carbon/human/procs/get_desc.dm

```diff
@@ -410,6 +410,29 @@
 		items = copytext(items, 3)
 		. += "<br>[SPAN_NOTICE("[src] is juggling [items]!")]"
 
+	if (src.reagents.has_reagent("ethanol"))
+		var/et_amt = src.reagents.get_reagent_amount("ethanol")
+		var/drunk_assess = ""
+		if (!isalcoholresistant(src) || src.reagents.has_reagent("moonshine"))
+			if (src.reagents.has_reagent("moonshine"))
+				et_amt *= 7
```

Comment:
I'd drop this part, since that's not *really* how moonshine works anyways - it increases the efficacy of ethanol effects, but doesn't affect the volume thresholds at all

Reply:
I see! I did fundamentally misunderstand how moonshine works, will revert